### PR TITLE
pin spectacle version

### PR DESCRIPTION
**Pull request by michaelmerrill**
the spectacle repo has already been edited to publish the docs/ directory. [spectacle/.npmignore](https://github.com/FormidableLabs/spectacle/blob/master/.npmignore)

### CODE REVIEW

**Comment by ryan-roemer** [comment]
Can we update this PR to also include this work https://github.com/FormidableLabs/formidable.com/issues/559 ? (Move everything to `devDependencies`). Thanks!

File: package.json [hunk 1]
```diff
@@ -21,27 +21,25 @@
...
+    "builder-docs-archetype-dev": "^4.2.2",
     "formidable-landers": "^8.3.2",
     "history": "^1.17.0",
     "lodash": "^4.16.4",
     "markdown-it": "^8.0.0",
     "markdown-it-toc-and-anchor": "^4.1.1",
     "marked": "^0.3.5",
     "normalize.css": "^4.2.0",
-    "prismjs": "^1.5.1",
+    "prismjs": "^1.6.0",
```
Comment (ryan-roemer) [inline package.json]: Pin this straight up:

```js
"prismjs": "1.6.0",
```

Comment (michaelmerrill) [reply package.json]: yep, done!

**Comment by michaelmerrill** [comment]
@ryan-roemer pinned the prismjs dep. Does this lgty? going to merge, publish, and update formidable.com

**Pull request merged**